src/module/item/ability/document.ts

```diff
@@ -70,7 +76,11 @@ class AbilityItemPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> exten
         options: DocumentModificationContext<TParent>,
         user: UserPF2e
     ): Promise<boolean | void> {
+        if (typeof changed.system?.category === "string") {
+            changed.system.category ||= null;
+        }
```

Comment:
```suggestion
        changed.system.category ||= null;
```

I believe that'll work?

Reply:
It wouldn't because `system` is possibly undefined